### DISCUSSION TITLE
Apartment door entry sytem

### DISCUSSION THREAD

**Discussion by Theoriginalwilko** [RavenSystem/esp-homekit-devices#1930] [MEPLHAA Scripts]
Alright guys here’s the situation. I live in an apartment with a “dumb” intercom/door opening phone. I want to fit a 2ch ESP relay to do two things: 1: to be able to pulse a relay to trigger the “open door” button (basically mimicking me pressing the button to let people in) and 2: the other relay to be in-line with the speaker for the doorbell buzzer and act essentially as a “switch” that I can automate to be turned off at night (drunk people love to walk past and mash all the buttons at ungodly hours) anyone wizardy enough to figure out a code for that? These are my board choices:
![E4BE1304-A017-4CFA-8FE7-73AAEB594017](https://user-images.githubusercontent.com/121274895/209268943-d16d5983-2b96-4810-89cb-7ef3286489fd.jpeg)
![BE12FF96-FAF3-4F16-836A-FB47465C2A5C](https://user-images.githubusercontent.com/121274895/209268948-6ac7d590-f3cb-49a8-9521-c1ff068bbb13.jpeg)

**Comment by CryptonWind** [comment]
- First you have to find the board schema and find out exactly about all the GPIO pins. 
- After that find a solution to write the Homekit firmware for this specific board. This includes how to put it in the flash mode, etc.
- Later, after you manage to put it in your WIFI network, try different simple configurations for normal Homekit Switch, then add another one and so on.

Personally I know how to implement your requirements with two Sonoff SV, 2xBasic but I don't know exactly with this specific dual relay module.

I recommend you to start playing first with a simple Sonoff SV and see how it works then try to do the same steps to this one.
